packages/cli/src/commands/migrate/tasks/dependency-install.ts

```diff
@@ -3,6 +3,17 @@ import { ListrTask } from 'listr2';
 import { addDep } from '../../../utils';
 import type { MigrateCommandContext, MigrateCommandOptions } from '../../../types';
 
+export const REQUIRED_DEPENDENCIES = [
```

Comment:
<nerd>I would sort them, but it's fine :)</nerd>

Reply:
I like it, sorted 😄